packages/mermaid/src/mermaidAPI.ts

```diff
@@ -28,17 +28,9 @@ import type { MermaidConfig } from './config.type.js';
 import { evaluate } from './diagrams/common/common.js';
 import isEmpty from 'lodash-es/isEmpty.js';
 import { setA11yDiagramInfo, addSVGa11yTitleDescription } from './accessibility.js';
+import type { DiagramStyleClassDef } from './diagram-api/types.js';
 import { preprocessDiagram } from './preprocess.js';
 
-// diagram names that support classDef statements
```

Comment:
I think we should groom several issues to widen support for classDef to all relevant diagrams. The groomed issues should outline what needs to be don and how to doit to make it easier for contributors to get started.